recipe/meta.yaml

```diff
@@ -33,14 +32,14 @@ requirements:
     - click >=6.0,<7.0
     - requests >=2.8,<3.0
     - cchardet >=1.0,<2.0
-    - unicodecsv >=0.14,<1.0
+    - unicodecsv >=0.14,<2.0
     - ijson >=2.0,<3.0
     - jsonlines >=1.1,<2.0
-    - sqlalchemy >=1.1,<2.0
+    - sqlalchemy >=0.9.6,<2.0
     - linear-tsv >=1.0,<2.0
     - xlrd >=1.0,<2.0
-    - openpyxl >=2.4,<3.0
-    - datapackage-py <1.0
+    - openpyxl >=2.4,<2.5
+    - datapackage-py >=1.1.3,<2.0
```

Comment:
this is an optional dependency so we can remove it to get rid of the cycle

Reply:
Why `datapackage` was renamed to `datapackage-py`?